vllm/model_executor/model_loader/loader.py

```diff
@@ -1369,6 +1382,11 @@ def load_model(self, vllm_config: VllmConfig) -> nn.Module:
                 if quant_method is not None:
                     with device_loading_context(module, target_device):
                         quant_method.process_weights_after_loading(module)
+                if isinstance(module, Attention) and \
+                    hasattr(module, "process_weights_after_loading"):
+                    # When attention modules need to process weights after
+                    # currently only used by MLA
+                    module.process_weights_after_loading()
```

Comment:
Was this one intentionally skipped with adding the model_config.dtype?
```suggestion
                    module.process_weights_after_loading(
                            model_config.dtype)
```

Reply:
Not intentional